nginx/nginx.conf

```diff
@@ -44,10 +44,12 @@ http {
         ssl_certificate             /etc/nginx/private/cert.pem;
         ssl_certificate_key         /etc/nginx/private/key.pem;
         ssl_prefer_server_ciphers   on;
-        ssl_session_timeout         30m;
-        ssl_session_cache           shared:SSL:5m;
-        ssl_protocols               TLSv1.2 TLSv1.1 TLSv1;
-        ssl_ciphers                 ECDHE-RSA-AES256-SHA:DHE-RSA-AES256-SHA:DHE-RSA-CAMELLIA256-SHA:DHE-RSA-AES128-SHA:DES-CBC3-SHA:!EXP:!ADH:!aNULL;
+        ssl_session_timeout         10m;
+        ssl_session_cache           shared:SSL:10m;
+        ssl_protocols               TLSv1.2 TLSv1.3;
+        ssl_ciphers                 "EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH";
```

Comment:
I'm not super up on my ciphers, but is this a random list or specific?  I'd maybe [defer to Mozilla](https://ssl-config.mozilla.org/#server=nginx&version=1.19.6&config=intermediate&openssl=1.1.1g&guideline=5.7), but would want to be careful to not orphan any old-ish, but still `TLSv1.2` compatible browsers.

Reply:
Okay I am meeting you half way here. I have added the older ones we used to support to the current list.

Reply:
@henokgetachew - what is the logic for meeting us half way instead of going with the more [explicit Mozilla list](https://ssl-config.mozilla.org/#server=nginx&version=1.19.6&config=intermediate&openssl=1.1.1g&guideline=5.7)? While I don't feel super strongly, the whole point of this ticket is to update the ciphers so I figure we may as well dig into this a bit!

my advice is to follow Mozilla with:

`ssl_ciphers ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:DHE-RSA-CHACHA20-POLY1305;`

While we're in there, do `ssl_prefer_server_ciphers off;` as well.

Otherwise, if you have explicit reasons why this is a bad idea, please lemme know!

Reply:
I agree with @mrjones-plip here. I'm by no means an expert but the advice seems to be if you're dropping TLS 1.1 and below, then you should turn `ssl_prefer_server_ciphers` off because it lets the client pick their preferred cypher which can improve performance on the device. Because these two settings are related it makes sense to change the `ssl_prefer_server_ciphers` off at the same time.